src/lib/components/TableOfContents/index.js

```diff
@@ -27,6 +27,7 @@ class TableOfContents extends BaseStateElement {
   static get properties() {
     return {
       opened: {type: Boolean, reflect: true},
+      i18n: {type: Object},
```

Comment:
Is it worth documenting on each class what properties we expect here? This might become useful if we move this to infra repro.